core/src/prediction/LLCausalPredictionStrategy.cpp

```diff
@@ -92,8 +93,10 @@ std::vector<double> LLCausalPredictionStrategy::predict(
     for (size_t j = 0; j < num_variables; ++j){
       size_t current_predictor = linear_correction_variables[j];
       // X - x0 column
-      X(i,j+1) = test_data->get(sampleID, current_predictor)
-                 - original_data->get(index, current_predictor);
+      //X(i,j+1) = test_data->get(sampleID, current_predictor)
```

Comment:
It'd be good to remove this commented out line. Also, was the calculation here incorrect before, or does the sign turn out not to matter? If it was actually incorrect, it'd be good to make sure we have a test to cover the behavior.

Reply:
You can formulate local linear regression either way; I changed it to be consistent with local linear prediction strategy.

Reply:
Got it, thanks.